Integrations/integration-jamf.yml

```diff
@@ -30,213 +30,164 @@ configuration:
-            var current = currentCommand.translator[j];
-            var entry = {
-                Type: entryTypes.note,
-                Contents: result,
-                ContentsFormat: formats.json,
-            };
-            var currentContent = current.innerPath ? dq(result, current.innerPath) : result;
-            var translated = mapObjFunction(current.data) (current.countingDict ? toArray(currentContent) : currentContent);
-            entry.ReadableContentsFormat = formats.markdown;
-            entry.HumanReadable = tableToMarkdown(current.title,translated);
-            entry.EntryContext = {};
-            var context = createContext(translated);
-            entry.EntryContext[current.contextPath] = context;
-            entries.push(entry);
-        }
-    } else {
-        return result;
-    }
-    return entries;
-  type: javascript
+        try:
+            contents = requests.post(base_url + endpoint.format(computer_id=computer_id,passcode=passcode),auth=auth,headers=HEADERS).text
+            demisto.results(contents)
+            sys.exit(0)
+        except Exception, e:
+            demisto.results({ 'Type' : entryTypes['error'], 'ContentsFormat' : formats['text'], 'Contents' : "Unable to complete command.  Please check credentials and/or proxy settings\n" + str(e) })
+            sys.exit(0)
+
+    if demisto.command() == 'test-module':
+        try:
+            endpoint = "/JSSResource/users"
+            contents = requests.get(base_url + endpoint,auth=auth,headers=HEADERS).json()
+            demisto.results('ok')
+            sys.exit(0)
+        except Exception, e:
+            demisto.results({ 'Type' : entryTypes['error'], 'ContentsFormat' : formats['text'], 'Contents' : 'Unable to login.  Please check credentials and/or proxy settings.'})
+            sys.exit(0)
+
+    if demisto.command() == 'jamf-get-computers':
+        jamf_get_computers()
+
+    if demisto.command() == 'jamf-get-computers-match':
+        jamf_get_computers_match(demisto.args()['match'])
+
+    if demisto.command() == 'jamf-get-computer-hardware':
+        jamf_computers_hardware(demisto.args()['id'])
+
+    if demisto.command() == 'jamf-erase-device':
+        jamf_erase_device(demisto.args()['id'],demisto.args()['passcode'])
+  type: python
   commands:
   - name: jamf-get-computers
-    description: Get a list of managed computers
     arguments: []
     outputs:
     - contextPath: Jamf.Computers.ID
       description: Computer ID
     - contextPath: Jamf.Computers.Name
       description: Computer name
+    description: Get a list of managed computers
   - name: jamf-get-computers-match
-    description: Get a list of managed computers that matches inputs query
     arguments:
     - name: match
       required: true
-      description: Match criteria. e.g. to retrieve all computers associated with a user use $UserID
+      description: Match criteria. e.g. to retrieve all computers associated with
```

Comment:
Add example, link to docs which give an example of `match criteria`

Reply:
This taken from the docs
`Name, mac address, etc. to filter by. Match uses the same format as the general search in Jamf Pro. For instance, admin* can be used to match computer names that begin with admin`